components/Tabs2/tests/unit/MDCTabBarViewTests.m

```diff
@@ -17,12 +17,131 @@
 #import "MDCTabBarView.h"
 
 @interface MDCTabBarViewTests : XCTestCase
+
+@property(nonatomic, strong) MDCTabBarView *tabBarView;
+
+@property(nonatomic, strong) UITabBarItem *itemA;
+
+@property(nonatomic, strong) UITabBarItem *itemB;
+
+@property(nonatomic, strong) UITabBarItem *itemC;
+
 @end
 
 @implementation MDCTabBarViewTests
 
+- (void)setUp {
+  [super setUp];
+
+  _tabBarView = [[MDCTabBarView alloc] init];
+  _itemA = [[UITabBarItem alloc] initWithTitle:@"A" image:nil tag:0];
+  _itemB = [[UITabBarItem alloc] initWithTitle:@"B" image:nil tag:0];
+  _itemC = [[UITabBarItem alloc] initWithTitle:@"C" image:nil tag:0];
+}
+
+- (void)tearDown {
+  _itemA = nil;
+  _itemB = nil;
+  _itemC = nil;
+  _tabBarView = nil;
+
+  [super tearDown];
+}
+
 - (void)testInitCreatesObject {
-  XCTAssertNotNil([[MDCTabBarView alloc] init]);
+  // Given
+  MDCTabBarView *tabBarView = [[MDCTabBarView alloc] init];
+
+  // Then
+  XCTAssertNotNil(tabBarView);
+  XCTAssertNotNil(tabBarView.items);
+}
+
+/// Tab bars should by default select nil in their items array. The behavior should also consistent
```

Comment:
Nit: s/also consistent/also be consistent/

Reply:
Fix applied.